client/src/components/user-admin/EditUsers.js

```diff
@@ -1,14 +1,16 @@
 import React, { useEffect, useState } from 'react';
 import '../../sass/UserAdmin.scss';
+import { FormGroup, FormControlLabel, Switch } from '@mui/material'
 
 // child of UserAdmin. Displays form to update users.
-const EditUsers = ({ userToEdit, backToSearch, updateUserDb, projects }) => {
+const EditUsers = ({ userToEdit, backToSearch, updateUserDb, projects, updateUserActiveStatus }) => {
   const [userManagedProjects, setUserManagedProjects] = useState([]); //  The projects that the selected user is assigned
   const [projectValue, setProjectValue] = useState(''); // State and handler for form in EditUsers
 
   // Prepare data for display
   const userName = `${userToEdit.name?.firstName} ${userToEdit.name?.lastName}`;
   const userEmail = userToEdit.email;
+  const [isActive, setIsActive] = useState(userToEdit.isActive);
```

Comment:
Maybe consider moving the state out of the "prepare data for display" section and putting it up next to the other `useState` instantiations